app/actions/deployment_create.rb

```diff
@@ -168,7 +169,8 @@ def deployment_for_stopped_app(app, message, previous_deployment, previous_dropl
           original_web_process_instance_count: desired_instances(app.oldest_web_process, previous_deployment),
           revision_guid: revision&.guid,
           revision_version: revision&.version,
-          strategy: DeploymentModel::ROLLING_STRATEGY
+          strategy: DeploymentModel::ROLLING_STRATEGY,
+          max_in_flight: message.options ? message.options[:max_in_flight] : 1
```

Comment:
**todo** We need to handle when options is set to an object, but the `max_in_flight` key is not set. I believe this would currently evaluate to null